ansible/README.rst

```diff
@@ -0,0 +1,30 @@
+Ansible and Docker OVN Emulation Guide
+======================================
+
+Overview
+--------
+
+Deploy an OVN emulation environment using Docker container and ansible
```

Comment:
Once deployed with ansible, it is supposed to setup rally environment like

```
rally deployment create --file=ovn-emulation.json --name=ovn-emulation
```
So once this PR got cleaned up, I will send another PR that extends rally to understand the OVN emulation environment. Of course, a large portion of this has done by the rally modified by you.